client/ayon_houdini/hooks/set_paths.py

```diff
@@ -15,4 +17,6 @@ def execute(self):
             self.log.warning("BUG: Workdir is not filled.")
             return
 
+        addon_dir = os.path.dirname(os.path.dirname(__file__))
```

Comment:
Unrelated change. Can be removed? Please revert changes to this file?

Reply:
done by copy pasting the original file back in from develop.